packages/e2e/setup/routing.ts

```diff
@@ -0,0 +1,20 @@
+import { configuration } from "./configuration";
+
+export const appUrls = (appUrl: string) => ({
+  manifest: new URL("/api/manifest", appUrl).href,
+  register: new URL("/api/register", appUrl).href,
+});
+
+export const saleorUrls = (dashboardUrl: string) => ({
```

Comment:
```suggestion
const saleorUrls = (dashboardUrl: string) => ({
```

nitpick, but I don't think it has to be exported